src/mpc-hc/MainFrm.cpp

```diff
@@ -11917,6 +11917,9 @@ HRESULT CMainFrame::PreviewWindowHide() {
 
         m_wndPreView.ShowWindow(SW_HIDE);
         m_wndPreView.SetWindowPos(&wndNoTopMost, 0, 0, 0, 0, SWP_NOMOVE | SWP_NOSIZE | SWP_NOACTIVATE);
+        if (AfxGetAppSettings().iOnTop) {
+            SetWindowPos(&wndTopMost, 0, 0, 0, 0, SWP_NOMOVE | SWP_NOSIZE | SWP_NOACTIVATE);
+        }

```

Comment:
Is there a specific reason to call SetWindowPos 2 times or am I missing something?

If not, isn't this more cleaner?
`m_wndPreView.SetWindowPos((AfxGetAppSettings().iOnTop? &wndTopMost : &wndNoTopMost), 0, 0, 0, 0, SWP_NOMOVE | SWP_NOSIZE | SWP_NOACTIVATE);`

Reply:
The second call refers to the main window, and not the preview window.  It restores it to topmost when hiding the preview window, because showing the preview window makes it lose topmost status.